src/euphorie/content/utils.py

```diff
@@ -426,3 +426,52 @@ def __call__(self):
         )
         response.setHeader("Content-Type", "text/csv;charset=utf-8")
         return csv_data
+
+
+def parse_scaled_answers(contents):
+    """Get values and answers from a scaled_answers risk field.
+
+    You pass the contents of the field: a multi-line string.
+
+    In most cases we will get something like this:
+
+        very unsafe
+        a bit unsafe
+        safe enough
+        quite safe
+        very safe
+
+    The first answer gets value 1, the second 2, etc.
+    But we also support something like this, with the values encoded:
+
+        very safe|5
+        quite safe|4
+        safe enough|3
+        a bit unsafe|2
+        very unsafe|1
+
+    """
+    result = []
+    if not contents:
+        return result
+    count = 0
+    for answer in contents.splitlines():
+        answer = answer.strip()
+        if not answer:
+            continue
+        count += 1
+        parts = answer.split("|")
```

Comment:
I think you want to use `.rpartition` here

Reply:
Done, looks simpler indeed.

Reply:
Please look at this https://github.com/euphorie/Euphorie/pull/717, it also cover another edge case